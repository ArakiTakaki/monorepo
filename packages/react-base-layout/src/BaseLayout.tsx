import * as styles from './BaseLayout.module.scss';
import { ImageComponent } from './ImageComponent';
import image from './example.png';

export const BaseLayout = () => {
  return (
    <div className={styles.baseLayout}>
      hello component <ImageComponent src={image} width={300} alt="alt" />
    </div>
  );
};
