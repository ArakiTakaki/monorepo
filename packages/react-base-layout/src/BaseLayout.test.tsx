import React from 'react';
import { render } from '@testing-library/react';
import { expect, test } from 'vitest';
import { BaseLayout } from './BaseLayout';

test('index page component', async () => {
  const { asFragment } = render(<BaseLayout />);
  expect(asFragment()).toMatchSnapshot();
});
