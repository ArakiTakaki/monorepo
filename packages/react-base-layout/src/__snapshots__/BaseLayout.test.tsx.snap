// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index page component 1`] = `
<DocumentFragment>
  <div>
    hello component 
    <img
      alt="alt"
      data-nimg="1"
      decoding="async"
      loading="lazy"
      src="/src/example.png"
      style="color: transparent;"
      width="300"
    />
  </div>
</DocumentFragment>
`;
